proxstar/__init__.py

```diff
@@ -175,7 +178,7 @@ def hostname(name):
 @auth.oidc_auth
 def vm_details(vmid):
     user = User(session['userinfo']['preferred_username'])
-    proxmox = connect_proxmox()
+    connect_proxmox()
```

Comment:
Not sure we even need these calls, I think these were just left behind from when I moved a bunch of the logic into the user/vm classes. If we're not doing anything with the result of `connect_proxmox`, no need for it.

Reply:
IMO that's enough of a deviation from this PR to warrant its own. I can go through all the code at a later date and remove any orphaned calls like this one.